### ISSUE TITLE
Dont work at Bittrex

### ISSUE THREAD

**Issue by flfsbr** [kelvinau/crypto-arbitrage#5]
This bot still working? For me only show this:

crypto-arbitrage$ sudo python main.py -m triangular -p
20180209011813 starting Triangular Arbitrage Engine...
checking open orders...
no open orders
starting to check order book...

And nothing more after this.

**Comment by kelvinau** [comment]
Yes it does. It's just that in production mode, it's not printing the debug checking message because I think it will be hard to find the message of the actual trade among all the others. It will only print if there's a possibly profitable trade.

**Comment by flfsbr** [comment]
Thanks for the answer. Can you indicate which are the best pairs for this type of operation? Or some tool that scans all the pairs and indicates the ones with the best opportunities?

**Comment by kelvinau** [comment]
Not that I know of any tools for that, probably because of the difficulties from the large number of combination pairs and the limited API requests required to get the information. I personally traded BTC-ETH-LTC before. Arbitrage requires large volume and these big cryptos usually have it. Of course, the exchange needs to support trading between the three choices.

**Comment by flfsbr** [comment]
Using BTC-ETH-LTC. Any idea?

20180210092054 starting Triangular Arbitra
checking open orders...
no open orders
starting to check order book...
[<Response [200]>, None, <Response [200]>]

[None]

[<Response [200]>, None, <Response [200]>]

[None, <Response [200]>, <Response [200]>]

[<Response [200]>, <Response [200]>, None]

[None]

[<Response [200]>, None, <Response [200]>]

[<Response [200]>, <Response [200]>, None]

[None, <Response [200]>, <Response [200]>]

[None, <Response [200]>, <Response [200]>]

[None]

[None, <Response [200]>, <Response [200]>]

[<Response [200]>, None, <Response [200]>]

[None, <Response [200]>, <Response [200]>]

[None, <Response [200]>, <Response [200]>]

[<Response [200]>, None, <Response [200]>]

[<Response [200]>, <Response [200]>, None]

[<Response [200]>, <Response [200]>, None]

[<Response [200]>, <Response [200]>, None]

[<Response [200]>, None, <Response [200]>]

[None, <Response [200]>, None]

[None]

[None]

[None, <Response [200]>, None]

[None]

[<Response [200]>, None, <Response [200]>]

[<Response [200]>, None, <Response [200]>]

[<Response [200]>, None, <Response [200]>]

[None, <Response [200]>, <Response [200]>]

[None, <Response [200]>, <Response [200]>]

[<Response [200]>, <Response [200]>, None]

[None, <Response [200]>, <Response [200]>]

[None, <Response [200]>, <Response [200]>]

[<Response [200]>, <Response [200]>, None]

[<Response [200]>, None, <Response [200]>]

[None, <Response [200]>, <Response [200]>]

**Comment by kelvinau** [comment]
This usually happens when Bittrex's server (and also the others) is not stable. The three responses are checking the price of the currencies, and you can see they return 200 separately. The arbitrage checking will only continue when all three responses return 200. You can wait for a bit first and try to run it again later.

**Comment by addjon** [comment]
Hello. Thank you for this project. I wanted to try out bot on Kraken exchange, but constantly get the answer [None] at the stage of checking balance. I can't figure out why the request doesn't get any response and how to deal with it?

**Comment by sanitariu** [comment]
Do you need to keep your money in USDT or in BTC ?
